apps/www/components/Nav/Developers.tsx

```diff
@@ -37,13 +38,11 @@ const Developers = () => {
       </div>
     )
     return url ? (
-      <a
-        key={text}
-        href={url}
-        className="dark:hover:bg-scale-500 col-span-6 rounded p-3 transition hover:bg-gray-50"
-      >
-        {content}
-      </a>
+      <Link href={url} key={text}>
```

Comment:
might need to use `passHref` here

Reply:
https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag